README.md

```diff
@@ -41,6 +41,8 @@ To configure the connection, have a look at SonarLint in default user settings.
 
 If you change something on the server such as the quality profile, you can trigger an update of the local cache using the "Update SonarLint binding to SonarQube/SonarCloud" command on the command palette (search for "sonarlint").
 
+When several sub-folders of the same server project are open in a single workspace, the project binding can be set at the workspace level.
```

Comment:
I guess we never really recommend to configure the binding at _User_ level.
What about saying  something like this?
'Configuring the project binding at workspace level mutes Won’t Fix or False Positive issues in any of the sub-folders you add to the workspace.'

I would then suggest to write it just after:
'To configure the connection, have a look at SonarLint in default user settings.'